scripts/certs/make_certs.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+set -e
+
+out_dir="/shared/cert"
+
+script_dir="$(dirname "$0")"
+mkdir -p "$out_dir"
+
+# list all certificates we want, so that we can check if they already exist
+server="/shared/cert/ca.crt /shared/cert/server.crt /shared/cert/server.key"
+client="/shared/cert/client.crt /shared/cert/client.key"
+targets="$client $server"
+
+echo ""
+echo "Checking certificates"
+recreate="false"
+# check if certificates exist
+for target in $targets; do
```

Comment:
```suggestion
set -- "$client" "$server"

echo ""
echo "Checking certificates"
recreate="false"
# check if certificates exist
for target do
```

Don't use a string as a list. With `sh`, set the positional parameters and then iterate over these in the loop.